src/CommunityToolkit.Maui/Converters/DateTimeOffsetConverter.shared.cs

```diff
@@ -43,7 +43,7 @@ public override DateTimeOffset ConvertBackTo(DateTime value, CultureInfo? cultur
 		};
 
 		return culture is null ?
-			new DateTimeOffset(value, offset) :
+			value.Kind == DateTimeKind.Local ? new DateTimeOffset(value) : new DateTimeOffset(value, offset) :
 			new DateTimeOffset(value.Year, value.Month, value.Day, value.Hour, value.Minute, value.Second, value.Millisecond, culture.Calendar, offset);
```

Comment:
Not really related to this change, or maybe it is... But this is not really readable anymore imho 😅 took me a good time to see what was actually going on here.

Reply:
we cannot use offset if dateTime.Kind is local. It may be different and cause tests to fail.